pkg/fftm/route_get_address_balance.go

```diff
@@ -0,0 +1,46 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fftm
+
+import (
+	"net/http"
+
+	"github.com/hyperledger/firefly-common/pkg/ffapi"
+	"github.com/hyperledger/firefly-transaction-manager/internal/tmmsgs"
+	"github.com/hyperledger/firefly-transaction-manager/pkg/apitypes"
+)
+
+var getAddressBalance = func(m *manager) *ffapi.Route {
+	return &ffapi.Route{
+		Name:   "getBalance",
+		Path:   "/gastoken/balances/{address}",
+		Method: http.MethodGet,
+		PathParams: []*ffapi.PathParam{
+			{Name: "address", Description: tmmsgs.APIParamSignerAddress},
+		},
+		QueryParams: []*ffapi.QueryParam{
+			{Name: "blockTag", Description: tmmsgs.APIParamBlockTag},
```

Comment:
Tiny point, but while these are case insensitive we've previously tried to make the docs the lower-case value.